base/string.jl

```diff
@@ -968,8 +968,8 @@ byte_string_classify(s::ByteString) = byte_string_classify(s.data)
     # 1: valid ASCII
     # 2: valid UTF-8
 
-is_valid_ascii(s::Union(Array{UInt8,1},ByteString)) = byte_string_classify(s) == 1
-is_valid_utf8(s::Union(Array{UInt8,1},ByteString)) = byte_string_classify(s) != 0
+isvalid(::Type{ASCIIString}, s::Union(Array{UInt8,1},ByteString)) = byte_string_classify(s) == 1
```

Comment:
Please calm down!
I wasn't threatening *ceasing* contributing - I said that I wouldn't have as much time, which is simply the facts.
I also did NOT say that people haven't been responsive in general to me... and in this very thread I thanked Tony and you for the thorough review.  I have been nothing *but* appreciative, in public and private, for the time people have spent giving constructive criticism, and have tried to respond as quickly as possible (and in fact, lost a lot of time due to Julia bugs, after having followed someone's suggestion to extend `convert` instead of having separate `utfx_to_utfy` functions)
Except for that issue, the code for #11004 hasn't changed in over a week.

I have been *donating* a lot of my time as well, trying to fix bugs in Julia, and solve some rather severe performance issues (whether or not you believe that string handling performance is important or not, is another issue).

I was just describing my situation... simply the facts... the client already gave up a week and a half ago on using Julia for a part of the project, because of issues with ODBC.jl being broken due to tupocalypse, string handling performance, and lack of decimal floating point support (even with DecFP.jl, that's very new, and is not integrated into JSON.jl nor ODBC.jl).
I've got a conference call first thing tomorrow morning, and I'm just worried that they will decide not to go ahead with using Julia for other parts of the project.
If they do (which is not up to me at all), then my involvement for now trying to improve Julia will necessarily drop down to what I can do in my spare time.  That's *NOT* an ultimatum, just the sad situation I'm in... and I've been doing everything in my power to prevent that happening.
I'd *much* prefer to do nothing but help make Julia the No. 1 language for string/database processing, than be programming in C++ and Python...

About the technical note:  by adding a new method to `convert`, would that be able to override all the code that uses the code in utf8.jl, utf16.jl, and utf32.jl for conversions?
I didn't realize that was possible - or even if possible, recommended (I've heard a lot about "monkey-patching" around here... a term I'd never come across before)